packages/client/src/features/tool-feedback/css-feedback.ts

```diff
@@ -23,16 +23,23 @@ export class ModifyCSSFeedbackAction implements Action {
     readonly elementIds?: string[];
 
     constructor(
-        public readonly elements?: SModelElement[],
+        public readonly input?: string[] | SModelElement[],
         public readonly addClasses?: string[],
         public readonly removeClasses?: string[],
         public kind = ModifyCssFeedbackCommand.KIND) {
-        if (elements) {
-            this.elementIds = elements.map(elt => elt.id);
+        if (input) {
+            this.elementIds = isStringArray(input) ? input : input.map(element => element.id);
         }
     }
 }
 
+function isStringArray(value: object): value is string[] {
```

Comment:
I think this method has high re-usability, do you think we can move it in a utility class somewhere?

Reply:
Done